rero_ils/modules/budgets/permissions.py

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8 -*-
+#
+# RERO ILS
+# Copyright (C) 2019 RERO
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, version 3 of the License.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""Budget permissions."""
+
+from ...permissions import staffer_is_authenticated
+
+
+def can_read_update_delete_budgets_factory(record, *args, **kwargs):
```

Comment:
**Cosmetic** : This method doesn't implement the 'read' permission rules. Either rename method, either update the code

Reply:
fixed